src/inset_state/inset_state.cpp

```diff
@@ -204,6 +234,18 @@ void InsetState::initialize_cum_proj()
   }
 }
 
+void InsetState::initialize_identity_proj()
+{
+  identity_proj_.resize(boost::extents[lx_][ly_]);
+  for (unsigned int i = 0; i < lx_; ++i) {
+    for (unsigned int j = 0; j < ly_; ++j) {
+      identity_proj_[i][j].x = i + 0.5;
+      identity_proj_[i][j].y = j + 0.5;
+    }
+  }
+  return;
```

Comment:
Clang-Tidy: Redundant return statement at the end of a function with a void return type